stock_vertical_lift_qty_by_packaging/models/vertical_lift_operation_base.py

```diff
@@ -11,7 +11,7 @@ class VerticalLiftOperationTransfer(models.AbstractModel):
         compute="_compute_product_qty_by_packaging_display"
     )
 
-    @api.depends("current_move_line_id.product_qty")
+    @api.depends("current_move_line_id.quantity")
```

Comment:
`product_qty` in 14.0 is now `quantity_product_uom` in 18.0. Quick summary:

| 14.0 | 18.0 | Description |
| ---- | ---- | ----------- |
| `product_qty` | `quantity_product_uom` | Qty in `<stock.move.line>.product_uom_id` (Order unit) |
| `product_uom_qty` | `quantity` | Qty in `<stock.move.line>.product_id.uom_id` (Stock unit, never change) |

Reply:
Well, maybe I was right at the end, sorry for the ping-pong comment.

Here is an attempt of migration data:

| 14.0 | 16.0 | 17.0 | 18.0 | Description |
| ---- | ---- | ---- | ---- | ----------- |
| `product_qty` | `reserved_qty` | `quantity_product_uom` | | Qty in `<stock.move.line>.product_uom_id` (Order unit) |
| `product_uom_qty` | `reserved_uom_qty` | `quantity` | | Qty in `<stock.move.line>.product_id.uom_id` (Stock unit, never change) |